components/ws-manager-bridge/src/bridge.ts

```diff
@@ -325,7 +325,7 @@ export class WorkspaceManagerBridge implements Disposable {
         // prebuilds are an EE feature - we just need the hook here
     }
 
-    protected async controlPrebuildInstance(instance: WorkspaceInstance): Promise<void> {
+    protected async stopPrebuildInstance(ctx: TraceContext, instance: WorkspaceInstance): Promise<void> {
```

Comment:
renaming, because this function is called in a single context, i.e. only when the status of a prebuild workspace instance is adjusted.